### PR TITLE
IOSXE/show lisp eid-table vrf {vrf} ipv4 database 

### PR DESCRIPTION
**Pull request by qduk**
## Description
IOSXE/show lisp eid-table vrf ipv4 databse

## Motivation and Context
Take unstructured data returned from command and turn it into structured data

## Screenshots:
Capture

## Checklist:
- [x] I have updated the changelog.
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

**Review by xiaoxinz-cisco — APPROVED** [review]
Thanks for your contribution!

**Review by tahigash — CHANGES_REQUESTED** [review]
reviewed schema and left 2 comment

File: src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfIpv4Database/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+expected_output = {
+    "vrf": "User",
+    "iid": 4100,
```
[the remark this answers is not in the document]
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfIpv4Database/cli/equal/golden_output_expected.py]: Updated

File: src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfIpv4Database/cli/equal/golden_output_expected.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "lsb": "0x3",
+    "total_entries": 3,
+    "no_route": 0,
+    "inactive": 0,
+    "eid": {
+        "10.16.0.0": {
```
[the remark this answers is not in the document]
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfIpv4Database/cli/equal/golden_output_expected.py]: Updated

**Review by tahigash — APPROVED** [review]
thanks for addressing comment. now schema looks good.

**Pull request merged**